lib/rucio/web/ui/flask/common/utils.py

```diff
@@ -360,7 +360,7 @@ def x509token_auth(data=None):
         msg = "<br><br>Your certificate (%s) is not mapped to (possibly any) rucio account: %s at VO: %s." % (html_escape(dn), html_escape(ui_account), html_escape(ui_vo))
     else:
         msg = "<br><br>Your certificate (%s) is not mapped to (possibly any) rucio account: %s." % (html_escape(dn), html_escape(ui_account))
-    msg += "<br><br><font color=\"red\">First, please make sure it is correctly registered in <a href=\"https://voms2.cern.ch:8443/voms/atlas\">VOMS</a> and be patient until it has been fully propagated through the system.</font>"
+    msg += "<br><br><font color=\"red\">First, please make sure it is correctly registered in <a href=\"https://voms2.cern.ch:8443/voms/atlas\">| VOMS |</a> and be patient until it has been fully propagated through the system.</font>"
```

Comment:
Need some more information regarding the configurable nature of the message 🤔

Reply:
I have added a quick comment to the issue about this #1109